File: api/src/main/java/com/qingketang/biz/classroom/entity/Classroom.java
package com.qingketang.biz.classroom.entity;

import lombok.Data;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;

import javax.persistence.*;

@Entity
@EntityListeners(AuditingEntityListener.class)
@Data
public class Classroom {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String no;

    private String name;

    private Long userNum;

    @CreatedDate
    private Long createdAt;

    @LastModifiedDate
    private Long updatedAt;
}

File: server/src/entity/Classroom.ts
import { Entity, Column, PrimaryGeneratedColumn } from "typeorm";

@Entity()
export class Classroom {
    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    no: string;

    @Column()
    name: string;

    @Column()
    user_num: number;

    @Column()
    created_at: number;

    @Column()
    updated_at: number;
}
